view/adminhtml/templates/menu/nodes.phtml

```diff
@@ -4,6 +4,18 @@ $fieldData = [];
 foreach ($block->getNodeForms() as $form) {
     $fieldData[$form->getNodeType()] = $form->getJsonConfig();
 }
+
+// TODO: backend - create method to return array of nodes components
+// https://github.com/SnowdogApps/magento2-menu/issues/69#issuecomment-443500749
+$vueComponents = [
```

Comment:
We can create some `Provider` service with default node types and end-users of the module can create plugin/interceptor for this `Provider` to merge own additional node types